server/src/main/java/org/opensearch/common/geo/GeoPoint.java

```diff
@@ -119,7 +119,11 @@ public GeoPoint resetFromString(String value, final boolean ignoreZValue, Effect
     public GeoPoint resetFromCoordinates(String value, final boolean ignoreZValue) {
         String[] vals = value.split(",");
         if (vals.length > 3) {
-            throw new OpenSearchParseException("failed to parse [{}], expected 2 or 3 coordinates " + "but found: [{}]", vals.length);
+            throw new OpenSearchParseException(
+                "failed to parse [{}], expected 2 or 3 coordinates " + "but found: [{}]",
+                vals.length,
+                vals.length
```

Comment:
isn't the second number redundant or it was a bug in original code?

Reply:
It was a bug in original code. Without the third parameter, you will see `"failed to parse [4], expected 2 or 3 coordinates " + "but found: [{}]`